CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## 4.0.4 2020-10-13
```

Comment:
This should probably be 4.1.0, because this is a new feature and not a bug fix.

Reply:
Fair enough. My think was that it was a bug not to include it the first time around